Adding beta enablement guide to docs (#6280)
PostHog/posthog.com: contents/docs/getting-started/enable-betas.mdx

---
title: Enabling beta features
isArticle: false
hideAnchor: true
---

> At PostHog, we ship _weirdly_ fast and there are always new betas for you to try. Many are  everyone by default, such as HogQL. Others require you to opt-in to try them out.

Reviewer: ```suggestion
At PostHog, we ship _weirdly_ fast and there are always new betas for you to try. Many are available to everyone by default, such as [HogQL](/docs/hogql). Others require you to opt-in to try them out.
```

You can follow along with [the latest betas and updates on our changelog](/changelog). If you're interested in joining an opt-in beta, here's how.

### Requirements

Opt-in betas are available to all PostHog Cloud users, as PostHog Cloud is automatically updated to run the latest version of PostHog. Betas are not available for Hobby deployments, or [sunset K8s deployments](/blog/sunsetting-helm-support-posthog). 

### How to join an opt-in beta

- Log in to your PostHog account

> - Select 'Enable beta features' from the dropdown menu

Reviewer: ```suggestion
- Select 'Enable beta features' from the account dropdown menu in the top right corner of the page
```

> - Select the betas you'd like to enable by enabling the toggles

Reviewer: ```suggestion
- Toggle the betas you'd like to enable
```

- That's it!

> 

Reviewer: ```suggestion
```

### Further reading

> You can always check the docs to get the latest information on PostHog features, or post here in the community to get advice. If you spot a bug or have feedback, [please let us know](http://app.posthog.com/home#supportModal)

Reviewer: ```suggestion
You can always check the [docs](/docs) to get the latest information on PostHog features, or post a [question in the community](/questions) to get advice. If you spot a bug or have feedback, [please let us know in-app](http://app.posthog.com/home#supportModal).
```
